src/dmd/backend/elem.d

```diff
@@ -999,47 +1012,52 @@ elem * el_una(uint op,tym_t ty,elem *e1)
  */
 
 extern (C) elem * el_longt(type *t,targ_llong val)
-{ elem *e;
-
-  assert(PARSER);
-  e = el_calloc();
-  e.Eoper = OPconst;
-  e.ET = t;
-  if (e.ET)
-  {     type_debug(t);
+{
+    elem *e;
+    assert(PARSER);
+    e = el_calloc();
```

Comment:
Could move the declaration to this line.

Reply:
There's a lot more that could be done, but this PR is long enough as it is.

Reply:
Fair enough. I just noticed you did a similar change in another place.